src/components/NFTEditionSelectItem.vue

```diff
@@ -93,6 +144,18 @@ export default {
       type: Boolean,
       default: false,
     },
+    discountedPrice: {
```

Comment:
Since we populate this field regardless of discount, call this `currentPrice`?

Reply:
[✏️ Improve props naming](https://github.com/likecoin/liker-land/pull/1544/commits/79e8cb189ad0145eee286c2acf71558a184ce5d8)